npm/webpack-preprocessor/package.json

```diff
@@ -22,14 +22,15 @@
   "dependencies": {
     "bluebird": "^3.7.1",
     "debug": "4.3.2",
-    "lodash": "^4.17.20"
+    "lodash": "^4.17.20",
+    "webpack": "^4.41.12"
```

Comment:
Just noticed this. webpack should be devDependency, since it's a peer dependency. We don't want to release it with a webpack included.